inference-engine/src/mkldnn_plugin/utils/general_utils.h

```diff
@@ -39,5 +40,21 @@ constexpr inline bool implication(bool cause, bool cond) {
     return !cause || !!cond;
 }
 
+inline std::string getExceptionDescWithoutStatus(const InferenceEngine::details::InferenceEngineException& ex) noexcept {
+    std::string desc = ex.what();
```

Comment:
M.b. I'm exaggerating the problem, but the std::string constructor may throw a bad_alloc exception in general case.

Reply:
Removed noexcept